DSCResources/MSFT_xDhcpServerAuthorization/MSFT_xDhcpServerAuthorization.psm1

```diff
@@ -0,0 +1,144 @@
+Import-Module $PSScriptRoot\..\Helper.psm1 -Verbose:$false
+
+# Localized messages
+data LocalizedData
+{
+    # culture="en-US"
+    ConvertFrom-StringData @'
+ResolvingIPv4Address = Resolving local IPv4 IP addresses
+ResolvingHostname    = Resolving local hostname
+AuthorizingServer    = Authorizing DHCP Server '{0}' with IP address '{1}'
+UnauthorizingServer  = Unauthorizing DHCP Server '{0}' with IP address '{1}'
+ServerIsAuthorized  = DHCP Server '{0}' with IP address '{1}' IS authorized
+ServerNotAuthorized  = DHCP Server '{0}' with IP address '{1}' is NOT authorized
+'@
+}
+
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        [Parameter(Mandatory)]
+        [ValidateSet('Present','Absent')]
+        [System.String] $Ensure,
+
+        [ValidateNotNullOrEmpty()]
+        [System.String] $DnsName = ( Get-Hostname ),
+
+        [ValidateNotNullOrEmpty()]
+        [System.String] $IPAddress = ( Get-IPv4Address | Select-Object -First 1 )
+    )
+    $targetResource = @{
+        DnsName = $DnsName;
+        IPAddress = $IPAddress;
+        Ensure = ''
+    }
+    if ($Ensure -eq 'Present') {
```

Comment:
Hehe :smile: Who indeed?! I actually tend to write braces the same way you do - but I found that PowerShell Studio 2015 actually has an automated change mechanism for this particular style guideline, but it tends to change other things at the same time that I don't want it to.

I'm sure it is possible to create a simple "fixer" for this and add it to the Meta.Fixers.psm1 module in DSCResource.Tests. I may put some time into looking into this when I get a chance.